packages/server/src/plugin/subscriptionCallback/index.ts

```diff
@@ -51,9 +51,7 @@ export function ApolloServerPluginSubscriptionCallback(
           return {
             http: {
               status: 200,
-              headers: new HeaderMap([
-                // ['content-type', 'application/json+graphql+callback/1.0'],
-              ]),
+              headers: new HeaderMap([['content-type', 'application/json']]),
```

Comment:
Shouldn't this one have the `callbackSpec=1.0` in it? This is the response to the original subscription request from the router.